README.md

```diff
@@ -1,16 +1,120 @@
-# Title
+# The Constant Expression
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Identify the _constant expression_
+* Explain how the _constant expression_ stops evaluation
+* Explain how the _constant expressoin_ creates agreement
 
 ## Introduction
 
-## SWBAT 1
+Our definition of _expression_ is:
 
-## SWBAT 2
+> An expression in a programming language is a combination of one or more
+> constants, variables, operators, ... that the programming language
+> interprets (according to its particular rules of precedence and of association)
+> and computes to produce ("to return", in a stateful environment) another value.
+> This process, as for mathematical expressions, is called evaluation.
+
+[Wikipedia][def]
+
+What if we were to make an expression that had no _variables_, since we haven't
+learned about them yet (but will soon!). What if it _also_ didn't have
+_operators_? That would be an expression like:
+
+```ruby
+42
+```
+
+This expression is the _constant expression_ and it's very important, although
+very boring From a programming as conversation perspective, the _constant
+expession_ has the power to create _agreement_ and _trust_. And if it weren't
+for you ability to agree with our conversational partner about basic reality,
+programming would never have been possible!
+
+## Define the _Constant Expression_
+
+The constant expression is an expression composed only of _constants_. Its
+_return value_ is the same as the constants.
+
+## Importance of the _Constant Expression_
+
+### Explain how the _constant expression_ stops evaluation
+
+This _expression_ is important because it tells Ruby _when to stop_.
+
+Let's consider a simple arithmetic expression. Keep in mind we apply operators
+in "[PEMDAS][]" order: parenthesis, exponents, multiplication, division,
+addition, subtraction.
+
+| Expression | Has Operators? | Operators | Are we done?| Which to Apply |
+| ---------- | -------------- | --------- | ------------| -------------- |
+| 3 * ( 10 - 5 ) | YES | `*`, `()`| NO | Open new sub-expression in `()` because of PEMDAS|
+| ( 10 - 5 )| YES | `-`| NO | `-` because of PEMDAS|
+| 5 | NO | NONE | YES| Constant expression! Return the value of the constant, we're done!|
+| 3 * 5 | YES | NO | Swap in the return value of the inner expression|
```

Comment:
doesn't this have operators (*)?

Reply:
Ughg, hard to edit on a laptop. improved.